internal/db/schema/migrations/oss/postgres/34/01_vault_library_map_user_password.up.sql

```diff
@@ -0,0 +1,6 @@
+begin;
+
+alter table credential_vault_library_user_password_mapping_override
+  rename to credential_vault_library_username_password_mapping_override; 
```

Comment:
Can the [comment on this table](https://github.com/hashicorp/boundary/blob/d4d81c24a6f79b3f5f12b86ce35fce2b1cab8208/internal/db/schema/migrations/oss/postgres/22/04_vault_library_map_user_password.up.sql#L20-L24) be updated to reflect the new name?